sympy/concrete/tests/test_products.py

```diff
@@ -348,6 +348,13 @@ def test_reverse_order():
            Product(x*y, (x, b + 1, a - 1), (y, 6, 1))
 
 
+def test_issue_9983():
+    n = Symbol('n', integer=True, positive=True)
+    p = Product(1 + 1/n**(S(2)/3), (n, 1, oo))
+    assert p.is_convergent() is S.false
+    assert p == p.doit()
```

Comment:
Particularly regarding this test, I think it would be good if you change the test as follows:
`product(1 + 1/n**(S(2)/3), (n, 1, oo)) = p.doit()`
The reason is similar to [this reason](https://github.com/sympy/sympy/pull/9919#discussion_r39552548)